src/lib/services/settings-service.ts

```diff
@@ -11,5 +12,6 @@ export const fetchSettings = async (request = fetch): Promise<Settings> => {
     auth: {
       enabled: !!settings?.Auth?.Enabled,
     },
+    defaultNamespace: settings?.DefaultNamespace || 'default',
```

Comment:
If you make the default in writable null or undefined then you can use the ?? operator and preserve strings but your code should be correct in general.

Reply:
addressed and made the goto navigation reactive until there is a value fetched